astrocut/cube_cut.py

```diff
@@ -46,10 +46,10 @@ def __init__(self, threads: Union[int, None] = None):
         table_data : `~astropy.io.fits.fitsrec.FITS_rec`
             The cube image header data table.
 
-        threads : int or `None`
+        threads : int or `None`, default=1
             Number of threads to use when making cutouts.
-            <=1 disables the threadpool, > 1 sets the number of threads,
-            None (default) uses Python's default threads
+            <=1 disables the threadpool, > 1 sets to the specified number of threads,
```

Comment:
Since the default is to not use the thread pool, can we schedule a backlog item to make it easy to find within the Astrocut documentation how a user enables this and why they may want to do so? @jaymedina

Reply:
I would prefer to have the documentation commits in a separate PR where we can review them separately.

Reply:
Sure thing @falkben , I've created a Jira ticket for this that I've moved to this sprint, so I'll tackle it this week and tag you when ready.